connectors/kafka/src/schemas/Test.avsc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Maybe an better solution is to generate these avro sources into the `test` folder and add the whole generated package to the connector level `.gitignore`.

For instance:
- Move the schemas into `src/test/resources/schemas` (seems more correct)
- Modify `build.sbt`:
```
(javaSource in avroConfig) := baseDirectory(_/"src/test/java").value
(sourceDirectory in avroConfig) := baseDirectory(_/"src/test/resources/schemas").value
```
- Add `/src/test/java/com/github/dcshock/avro/schemas` (maybe there's a better package name? `forklift.schemas`?) to the kafka-connector `.gitignore`.

Only real downside to this is if we want to use avro java objects in the main connector, it becomes difficult (although I'm sure it's solvable).